### PR TITLE
add aws outpost support

### PR DESCRIPTION
**Pull request by AsherShoshan**

### CODE REVIEW

**Comment by jianlinliu** [comment]
Several questions:
1. Pls always ensure the validation jobs get passed.
2. To be honest, I did not get the points of `OUTPOST` meaning? Can you explain more?

File: ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+chain:
+  as: ipi-conf-aws-outpost
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml:2]: If this chain only includes one ref, I guess we do not need to introduce the chain.

Comment (AsherShoshan) [reply ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml:2]: so just remove *-chain from folder?

to ur questions above:

Comment (jianlinliu) [reply ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml:2]: > 'outpost' is an on-premise hardware of Amazon
Sounds like need a specific aws account, right? If right, I think you need start from the very beginning of CLUSTER_PROFILE creation.

Comment (AsherShoshan) [reply ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-chain.yaml:2]: Why create a new cluster_profile, and not just inject my domain + credentials?  (it is going to be temp solution until 'regular' ci aws account will have it's own 'outpost'.  right?

**Pull request closed without merging**